web/packages/core/src/ruffle-player.ts

```diff
@@ -1401,6 +1401,7 @@ export class RufflePlayer extends HTMLElement {
             return;
         }
 
+        this.hidePreloader();
```

Comment:
Don't we want to place this call right at the beginning of the function? So even when the `if` above is taken, the preloader will be hidden.

Reply:
The preloader gets hidden in the panic function so so it doesn't need to be hidden twice.